### DISCUSSION TITLE
Allow the option to enable the respawn shuttle in campaign

### DISCUSSION THREAD

**Discussion by Arctem** [FakeFishGames/Barotrauma#4834] [Ideas and suggestions]
I understand why the respawn shuttle isn't enabled by default in the campaign, since it's sort of a rogue-like, but when missions can take a very long time to complete -- especially when things go wrong -- it's very miserable for someone who dies early on in a mission that the rest of the crew keeps trying to complete. I fully understand that death should result in losing any skill points and getting a new character, but players should be able to generate a new character and take a rescue shuttle back if they die.

I'd even be fine if it cost some money to do. Currently it's just a miserable experience when I'm playing with friends and we have to tell one person "well we'll be done in a few minutes...or maybe an hour. Who knows?"

Also this would allow for players that join the session mid-mission to join the submarine, which is a nice bonus.

**Comment by RadCatish** [comment]
I completely agree. Perhaps the respawn shuttle could set of from the nearest of the outposts to the sub (the start/endpoint) so that it stays immersive?

**Comment by NilanthAnimosus** [comment]
Some thoughts on this:

I do think this should be a campaign setting or even a default service - I would have it as something anyone whom can manage campaign can trigger and call for a submarine at cost. Where I am not sure - perhaps quite simply in the Crew/Mission/Character screen (Specifically under the crew list, as a button, showing both cost and current marks the campaign has to spend), this could also be used to perhaps manually trigger a set number of respawns in mission mode (? but this would take further thinking and expanding upon).

I think the cost should be somewhere between 500 marks to 2000 marks. which is a considerable hit on mission profits to use once, let alone twice.

it cannot respawn the AI. they cost considerable money per person, and should be for players convenience in long games to spawn your friends back in if the round may be very long (IE. wreck salvage).

sometimes there may be no nearby outpost on both sides of a given map you are in, so perhaps sometimes it will not be available - I think this would be a fair tradeoff for going in very deep into zones far more dangerous as a wipeout of your submarine becomes a more real threat throwing money at and more bodies cannot prevent.

The shuttle if available from the nearest outpost should not have a time limit and can come from right or left if needed - and only leave X time no player has been inside it, say 1 minute.

it also must not be kept docked on level end (Just auto undock the respawn shuttle when you 'enter' the next map so it does not get saved with the submarine nothing too fancy).

Anyone left on the shuttle at level end (IE. entering next area) should be considered as dead unless close enough to the outpost (Within 150 meters? 200?). they will respawn in the next round anyways.

Please have the shuttle have a reduced requirement to pilot, most players will inevitably have a very low helm skill usually under 10, and preferably have said shuttle uniquely have a reduced requirement, though not necessary it would just be convenient.

**Reply by Hinny** [reply]
This issue should be closed since it is already implemented.
